input/fsh/requirements/hiv-fxnreq.fsh

```diff
@@ -0,0 +1,180 @@
+Instance: hiv-fxnreq
+InstanceOf: Requirements
+Usage: #example
+Title: "HIV - Business Process E: Register Client - Functional requirements"
+Description: "HIV - Business Process E: Register Client - Functional requirements"
+* status = #active
+* statement[0]
+  * key = "HIV.FXNREQ.001"
+  * requirement = "As a [Health worker or data entry clerk](ActorDefinition-HealthWorker.html)\nClients’ rights and safety are protected and the facility is in compliance with any relevant international or local data protection policies."
```

Comment:
A few things here:

1. We shouldn't have links to the personas because an issue here is that the personas in the functional requirements do not correspond to the personas in the L2 narrative (also, "Health worker" is not a persona in the L2 narrative, where Data Entry Clerk is a separate persona).
2. The SOP requires we follow the structure given in the Excel, that is the basic structure of each requirement should be:
   As a...
   I want...
   So that...
3. Capitalization should read normally, like a text description, so (exactly like in the example):
   As a health worker or data entry clerk
   I want to ensure that the privacy and confidentiality of clients are assured
   So that client's rights and safety are protected and the facility is in compliance with any relevant international or local data protection policies
4. The `requirement` element is `markdown`, so it might be better to use FSH's multiline strings, so we can write:
  ```
  * requirement = """
     As a health worker or data entry clerk
     I want to ensure that the privacy and confidentiality of clients are assured
     So that client's rights and safety are protected and the facility is in compliance with any relevant international or local data protection policies
  """
  ```